ansible/library/port_alias.py

```diff
@@ -57,13 +57,26 @@ def get_portmap(self):
             raise Exception("Something wrong when trying to find the portmap file, either the hwsku is not available or file location is not correct")
         with open(self.filename) as f:
             lines = f.readlines()
-        for  line in lines:
-            if 'Ethernet' in line:
-                mapping = line.split()
-                if len(mapping) < 3:
-                    self.portmap.append(mapping[0])
-                else:
-                    self.portmap.append(mapping[2])
+        index = 0
+        if re.match('^#', lines[0].strip()):
```

Comment:
could you refer to this pull request and store all the information in the `port_config.ini` file accordingly?

Reply:
when building docker-sonic-mgmt, it will install the device package which include all port_config.ini files to the correct location in docker-sonic-mgmt. Is this what you want?

Reply:
sorry I didn't add the link to my previous comment: https://github.com/Azure/sonic-swss/pull/287/files this pull request has a better logic to parse the `port_config.ini` file.